dcat/index.html

```diff
@@ -780,7 +780,7 @@ <h3>Class: Catalog</h3>
         <table class="definition">
             <thead><tr><th>RDF Class:</th><th><a href="http://www.w3.org/ns/dcat#Catalog">dcat:Catalog</a></th></tr></thead>
             <tbody>
-            <tr><td class="prop">Definition:</td><td>A curated collection of metadata about datasets and data services</td></tr>
+            <tr><td class="prop">Definition:</td><td>A curated collection of metadata about cataloged resources (e.g., datasets and data services)</td></tr>
```

Comment:
IMO the proposal from Simon captures the nuance. I think here we are making more evident what we have already written in other parts of the document, for this reason, I do not see a real issue on the fact that this change should consider as purely editorial.  

```suggestion
            <tr><td class="prop">Definition:</td><td>A curated collection of metadata about cataloged resources (e.g., datasets and data services in the context of a data catalog)</td></tr>
```